crates/common/src/lib.rs

```diff
@@ -17,7 +17,10 @@ pub use sparse_merkle_tree;
 
 pub const DEPOSITION_CODE_HASH: [u8; 32] = [0u8; 32];
 pub const SUDT_CODE_HASH: [u8; 32] = [0u8; 32];
-pub const CKB_SUDT_SCRIPT_HASH: [u8; 32] = [0u8; 32];
+pub const CKB_SUDT_SCRIPT_HASH: [u8; 32] = [
```

Comment:
Where did this come from?

Reply:
It's a temp value to pass the `assert` in https://github.com/nervosnetwork/godwoken/blob/2c515f5d13fcec611b1fdbac558b48609e855e64/crates/store/src/genesis.rs#L50-L54